scalding-core/src/main/scala/com/twitter/scalding/BijectedOrderedSerialization.scala

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2014 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+package com.twitter.scalding
+
+import com.twitter.scalding.serialization.OrderedSerialization
+import com.twitter.bijection.Bijection
+
+object BijectedOrderedSerialization {
+  implicit def fromBijection[T, U](implicit bij: Bijection[T, U], ordSer: OrderedSerialization[U]) =
```

Comment:
when taking an implicit Bijection, we need to use the `ImplicitBijection` type because it handles the fact that `Bijection[U, T]` would also work here.